WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -58,26 +60,35 @@ public partial class SendViewModel : RoutableViewModel
 	[AutoNotify] private bool _conversionReversed;
 	[AutoNotify(SetterModifier = AccessModifier.Private)] private SuggestionLabelsViewModel _suggestionLabels;
 
-	public SendViewModel(UiContext uiContext, WalletViewModel walletVm)
+	public SendViewModel(UiContext uiContext, IWalletModel walletModel, SendFlowModel parameters)
 	{
 		UiContext = uiContext;
-		WalletVm = walletVm;
 		_to = "";
 
-		_wallet = walletVm.Wallet;
+		_wallet = parameters.Wallet;
+		_walletModel = walletModel;
+		_parameters = parameters;
 		_coinJoinManager = Services.HostedServices.GetOrDefault<CoinJoinManager>();
 
 		_conversionReversed = Services.UiConfig.SendAmountConversionReversed;
 
-		ExchangeRate = _wallet.Synchronizer.UsdExchangeRate;
+		ExchangeRate = _walletModel.AmountProvider.UsdExchangeRate;
 
-		Balance = walletVm.WalletModel.Balances;
+		Balance =
+			_parameters.IsManual
+			? Observable.Return(_walletModel.AmountProvider.Create(_parameters.AvailableAmount))
+			: _walletModel.Balances;
 
-		_suggestionLabels = new SuggestionLabelsViewModel(WalletVm.WalletModel, Intent.Send, 3);
+		BalanceCaption =
+			_parameters.IsManual
+			? "Max:"
+			: "Balance:";
```

Comment:
It can be always `Max` regardless of the mode.

Reply:
Fixed.